azerothcore/azerothcore-wotlk
# Fix(DB/Locale): Locale fix for quest 919

## CHANGES PROPOSED:
-  Change the text from 10 to 12 for locales.

Co-authored-by: Pangolp <pangolp@users.noreply.github.com>

## TESTS PERFORMED:
- Ran the queries, all works.




## HOW TO TEST THE CHANGES:

Before: With a client different from english, .add quest 919 and check the objectives, it says 10 when they are supposed to be 12.
After: Again with a client different from english and after you clear Cache, it says 12.


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
